src/Symfony/Component/ExpressionLanguage/ExpressionFunction.php

```diff
@@ -62,4 +62,43 @@ public function getEvaluator()
     {
         return $this->evaluator;
     }
+
+    /**
+     * Creates an ExpressionFunction from a PHP function name.
+     *
+     * @param string      $phpFunction        The PHP function name
+     * @param string|null $expressionFunction The expression function name (default: same than the PHP function name)
+     *
+     * @return self
+     *
+     * @throws \InvalidArgumentException if given PHP function name does not exist
+     * @throws \InvalidArgumentException if given PHP function name is in namespace
+     *                                   and expression function name is not defined
+     */
+    public static function fromPhp($phpFunction, $expressionFunction = null)
```

Comment:
I had to read the PHP doc to know what is  `expressionFunction`. I would rename it to `expressionFunctionName` or something else.